src/common/Pagination/styled.js

```diff
@@ -24,74 +21,63 @@ export const Button = styled.button`
   gap: 6px;
   transition: 0.5s;
   cursor: pointer;
-
-  &:hover {
+  &:hover:not(:disabled) {
     filter: brightness(103%);
   }
   &:disabled {
     background-color: ${({ theme }) => theme.colors.mystic};
+    cursor: not-allowed;
```

Comment:
I think it might be a little too scary for the user, default cursor will do just fine 👌

Reply:
done ;-)